docs/semantic-search.ipynb

```diff
@@ -391,9 +296,24 @@
         "## Upserting data into the Pinecone index"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Here, we embed and upsert the data into Pinecone. What this means is that each record we formatted above will interact with our embedding model we specified prior, and produce a vector embedding. Then, we take these embedding batches and store them in Pinecone with the additional information we specified, which is also known as metadata. \n",
+        "\n",
+        "Metadata is handy for things like filtering, like for if you stored several languages in the same index and want to return just one based on metadata. To learn more about metadata, take a [look here](https://docs.pinecone.io/guides/index-data/indexing-overview#metadata).\n",
+        "\n",
+        "We specify and create a namespace called \"english-sentences\", which is a higher level unit of organization when interacting with Pinecone. \n",
+        "\n",
+        "Querying on namespaces performs a sort of broad filter to only records that exist in that namespace, which has the nice effect of speeding up searches too. \n",
+        "\n",
+        "To learn more about namespaces, [look here](https://docs.pinecone.io/guides/index-data/indexing-overview#namespaces)\n"
```

Comment:
Something is weird about this link. It did not show up as clickable in colab, which is maybe expected. Instead, if it can't be clickable, include the url directly.

Reply:
Works locally.. I'll inspect it.